srcpkgs/python3-rss2email/template

```diff
@@ -1,16 +1,17 @@
 # Template file for 'python3-rss2email'
 pkgname=python3-rss2email
-version=3.10
+version=3.11
 revision=1
 archs=noarch
 wrksrc="rss2email-${version}"
 build_style=python3-module
 pycompile_module="rss2email"
 hostmakedepends="python3-setuptools"
 depends="python3-feedparser python3-html2text"
+checkdepends="${makedepends} python3-BeautifulSoup4"
```

Comment:
What's the point of adding the makedepends here when the variable is empty?

Reply:
I followed the suggestion in the previous PR. Fixed in 3b23002

Reply:
My mistake, depends should be used.

Reply:
@Chocimier: fixed in 4ea6937